app/scripts/vendors/dimVendorItems.directive.js

```diff
@@ -38,7 +38,7 @@
       '      <img class="vendor-icon" ng-src="{{::vendor.icon | bungieIcon}}" />',
       '      {{::vendor.name}}',
       '    </div>',
-      '    <timer class="vendor-timer" ng-if="vendor.nextRefreshDate[0] !== \'9\'" end-time="vendor.nextRefreshDate" max-time-unit="\'day\'" interval="1000">{{days}} day{{daysS}} {{hhours}}:{{mminutes}}:{{sseconds}}</timer>',
+      '    <timer class="vendor-timer" ng-if="vendor.nextRefreshDate[0] !== \'9\'" end-time="vendor.nextRefreshDate" max-time-unit="\'day\'" interval="1000">{{days}} <span ng-if="days === 1" translate="Vendors.Day"></span><span ng-if="days !== 1" translate="Vendors.Days"></span> {{hhours}}:{{mminutes}}:{{sseconds}}</timer>',
```

Comment:
The translate plugin can handle plurals, I think. Let's use this as a test-bed for that.

I also think angular-timer supports localization more directly.